Samples~/AvatarCreatorSamples/AvatarCreatorWizard/Scripts/UI/ReloadPreviouslyEditedAvatar.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using ReadyPlayerMe.AvatarCreator;
+using ReadyPlayerMe.Core;
+using UnityEngine;
+using UnityEngine.Events;
+using UnityEngine.UI;
+
+namespace ReadyPlayerMe.Samples.AvatarCreatorWizard
+{
+    public class ReloadPreviouslyEditedAvatar : MonoBehaviour
+    {
+        private const string TAG = nameof(ReloadPreviouslyEditedAvatar);
+
+        [SerializeField] private Button continueEditingAvatar;
+        [SerializeField] private Button cancelEditingAvatar;
+
+        private string avatarId;
+        public UnityEvent<string> onSaveAvatar;
+        public UnityEvent onCancel;
+        public async void Show(string avatarId)
+        {
+            this.avatarId = avatarId;
+            try
+            {
+                await new AvatarAPIRequests().GetAvatar(avatarId, true);
+                gameObject.SetActive(true);
+            }
+            catch (Exception e)
+            {
+                OnCancelEditing();
+                SDKLogger.Log(TAG, $"Failed to load draft avatar with id ${avatarId}. {e.Message}");
+            }
+        }
+        private void OnEnable()
+        {
+            continueEditingAvatar.onClick.AddListener(OnContinueEditing);
+            cancelEditingAvatar.onClick.AddListener(OnCancelEditing);
+        }
+        private void OnDisable()
```

Comment:
You can run Riders code reformat and it will fix the spacing between functions. The functions should have 1 space in between each.